### PR TITLE
Remove unnecessary util.callable usage

### LINKED ISSUE #4850: util.callable is no longer necessary
`util.callable` is a compatibility helper to support Python 3.2. Since Python 3.2 is no longer supported, it is no longer necessary.

https://github.com/sqlalchemy/sqlalchemy/blob/17b7b3027f0e994cf47bf5fa3e536d3971076dcd/lib/sqlalchemy/util/compat.py#L161-L166

### PR DESCRIPTION
**Pull request by sloria**
Fixes: #4850

### Description

Removes usage of `util.callable`.

### Checklist

This pull request is:

- [ ] A documentation / typographical error fix
	- Good to go, no issue or tests are needed
- [x] A short code fix
	- please include the issue number, and create an issue if none exists, which
	  must include a complete example of the issue.  one line code fixes without an
	  issue and demonstration will not be accepted.
	- Please include: `Fixes: #<issue number>` in the commit message
	- please include tests.   one line code fixes without tests will not be accepted.
- [ ] A new feature implementation
	- please include the issue number, and create an issue if none exists, which must
	  include a complete example of how the feature would look.
	- Please include: `Fixes: #<issue number>` in the commit message
	- please include tests.

**Have a nice day!**

### CODE REVIEW

File: lib/sqlalchemy/util/compat.py [hunk 1]
```diff
@@ -158,14 +158,8 @@ def u(s):
...
-    if py32:
-        callable = callable  # noqa
-    else:
-
-        def callable(fn):  # noqa
-            return hasattr(fn, "__call__")
-
-
+    # Unused. Kept for backwards compatibility.
+    callable = callable  # noqa
```
Comment (zzzeek) [inline lib/sqlalchemy/util/compat.py]: good idea

**Review by sqla-tester — COMMENTED** [review]
OK, this is **sqla-tester** setting up my work to try to get revision a0ccdff2cb74f5e944d8baccc269c382b591c8e2 of this pull request into gerrit so we can run tests and reviews and stuff

**Comment by sqla-tester** [comment]
New Gerrit review created for change a0ccdff2cb74f5e944d8baccc269c382b591c8e2: https://gerrit.sqlalchemy.org/#/c/sqlalchemy/sqlalchemy/+/1459

**Comment by sqla-tester** [comment]
**mike bayer** (zzzeek) wrote:

Code-Review+2 Workflow+1

looks great thanks!

**Comment by sqla-tester** [comment]
Gerrit review https://gerrit.sqlalchemy.org/1459 has been **merged**. Congratulations! :)

**Pull request closed without merging**